src/pages/index.js

```diff
@@ -73,14 +110,38 @@ const createCard = (data) => {
   ).generateCard();
   return cardElement;
 };
+//
+userInfo.fetchAndSetUserInfo();
 
+api.getInitialCards().then((initialCards) => {
+  gallery = new Section(
+    {
+      data: initialCards,
+      renderer: (item) => {
+        const cardElement = createCard(item);
+        gallery.addItem(`append`, cardElement);
+      },
+    },
+    ".gallery__list"
+  );
+  gallery.renderItems();
+});
```

Comment:
You need to `catch` possible errors at the very **end** of any server request.

**Please, correct it everywhere in the code**